integrationtests/tests/tests.rs

```diff
@@ -408,10 +408,10 @@ async fn receive_lightning_payment_invalid_preimage() {
         .await;
     assert!(response.is_err());
 
-    // Gateway re-claims their funds
+    // Gateway is refunded
     let _outpoint = gateway
         .client
-        .claim_incoming_contract(contract_id, rng())
+        .refund_incoming_contract(contract_id, rng())
```

Comment:
Had to rename this because compiler didn't like having the same function defined for `Client<UserClientConfig>` and `Client<GatewayClientConfig>`.